### PR TITLE
Update case docstring to support polarion upload

### PR DESCRIPTION
**Pull request by Junefen**
- Move test files `test_hypervisors_state.py` and `test_hypervisors_sw.py` to another new directory tests/others, because the two files are not the function testing and no need to upload testcases to polarion.
- Will add detail comments to the code line
- Have tested locally to upload the cases to polarion successfully.

### CODE REVIEW

File: tests/function/test_config.py [hunk 1]
```diff
@@ -125,7 +125,7 @@ def test_print_in_virtwho_conf(
     ):
         """Test the print_ option in /etc/virtwho.conf
 
-        :title: virt-who: config: test print_ option (positive)
+        :title: virt-who: config: test print option (positive)
```
Comment (Junefen) [inline tests/function/test_config.py:128]: `print_` is not recoganized well by the polarion, it always show abnormally.

File: tests/function/test_service.py [hunk 2]
```diff
@@ -178,8 +178,8 @@ def test_virtwho_and_rhsmcertd_servce(self, virtwho, ssh_host):
     def test_virtwho_ignore_swp_file(self, virtwho, ssh_host):
         """
 
-        :title: virt-who: service: test virt-who service ignores
-            /etc/virt-who.d/*.swp file
+        :title: virt-who: service: test virt-who service ignores /etc/virt-who.d/xx.swp
```
Comment (Junefen) [inline tests/function/test_service.py:181]: `*` is not recognized by the polarion, will fail.

File: tests/hypervisor/test_ahv.py [hunk 3]
```diff
@@ -175,14 +175,19 @@ def test_prism_central(self):
...
-        pass
+        assert True
 
     @pytest.mark.tier1
     def test_fake_type(self, virtwho, function_hypervisor, hypervisor_data):
         """Test the fake type in /etc/virt-who.d/hypervisor.conf
 
         :title: virt-who: ahv: test fake type
         :id: 88212bd7-c5d6-46cc-a62b-4f06599ff71d
+        :caseimportance: High
```
Comment (Junefen) [inline tests/hypervisor/test_ahv.py:186]: missing

File: tests/hypervisor/test_esx.py [hunk 4]
```diff
@@ -370,7 +370,7 @@ def test_type(self, virtwho, function_hypervisor, esx_assertion):
         """Test the type= option in /etc/virt-who.d/test_esx.conf
 
         :title: virt-who: esx: test type option
-        :id: 0f2fcf4f-ac60-4c3c-905a-fabe22536ab2
+        :id: abc67f6f-c79d-4458-9c69-ac788f0697a6
```
Comment (Junefen) [inline tests/hypervisor/test_esx.py:373]: id conflict

File: tests/hypervisor/test_esx.py [hunk 5]
```diff
@@ -870,6 +870,7 @@ def test_filter_host_parents(self, virtwho, function_hypervisor, hypervisor_data
         :tags: tier1
         :customerscenario: false
         :upstream: no
+        :steps:
```
Comment (Junefen) [inline tests/hypervisor/test_esx.py:873]: missing

File: tests/hypervisor/test_hyperv.py [hunk 6]
```diff
@@ -160,13 +160,19 @@ def test_exclude_hosts(self, virtwho, function_hypervisor, hypervisor_data):
...
 
+    @pytest.mark.tier1
     def test_fake_type(
         self, virtwho, function_hypervisor, hypervisor_data, function_host_register
     ):
         """Test the fake type in /etc/virt-who.d/hypervisor.conf
 
         :title: virt-who: hyperv: test fake type
         :id: a2c30d57-4eae-451e-b131-2547f046463c
+        :caseimportance: High
```
Comment (Junefen) [inline tests/hypervisor/test_hyperv.py:171]: id conflict

File: tests/hypervisor/test_multi_hypervisors.py [hunk 7]
```diff
@@ -13,15 +13,16 @@ class TestMultiHypervisors:
...
-        :title: virt-who: base: test multi hypervisors report together
+        :title: virt-who: multiHypervisors: test multi hypervisors report together
         :id: 9fb6694e-7535-4d9f-9ac0-75e6cbe3066d
         :caseimportance: High
         :tags: multiHypervisor
         :customerscenario: false
         :upstream: no
         :steps:
-            1.
+            1. demo
```
Comment (Junefen) [inline tests/hypervisor/test_multi_hypervisors.py:23]: failed to upload the case when leave the value null

File: tests/hypervisor/test_xen.py [hunk 8]
```diff
@@ -22,11 +22,12 @@ def test_hostname_option(self):
         :customerscenario: false
         :upstream: no
         :steps:
-            1.
+            1. pending
```
Comment (Junefen) [inline tests/hypervisor/test_xen.py:25]: failed to upload the case when leave the value null

File: utils/polarion_testcase_upload.py [hunk 9]
```diff
@@ -35,8 +35,9 @@ def xml_file_generate():
         f"{args.project} "
         f"{args.xml_file}"
     )
-    logger.info(cmd)
-    ret, _ = subprocess.getstatusoutput(cmd)
+    logger.info(f"\n{cmd}\n")
```
Comment (Junefen) [inline utils/polarion_testcase_upload.py:38]: display the betelgeuse command clearly in the log to help debug.

File: virtwho/provision/virtwho_host.py [hunk 10]
```diff
@@ -54,6 +54,7 @@ def provision_virtwho_host(args):
 
     # Initially setup the virt-who host
     ssh_host.runcmd(cmd="rm -f /etc/yum.repos.d/*.repo")
+    ssh_host.runcmd(cmd="subscription-manager unregister; subscription-manager clean")
```
Comment (Junefen) [inline virtwho/provision/virtwho_host.py:57]: perfect the code format

File: virtwho/register.py [hunk 11]
```diff
@@ -81,7 +81,7 @@ def unregister(self):
         Unregister and clean host by subscription-manager.
         """
         ret, _ = self.ssh.runcmd(
-            "subscription-manager unregister;" "subscription-manager clean"
+            "subscription-manager unregister; subscription-manager clean"
```
Comment (Junefen) [inline virtwho/register.py:84]: perfect the code format

**Review by hkx303 — APPROVED** [review]
ACK

**Pull request merged**